Source/Scene/ShadowMap.js

```diff
@@ -135,6 +136,7 @@ define([
         //>>includeEnd('debug');
 
         this.enabled = false;
+        this._softShadows = defaultValue(options.softShadows, false);
```

Comment:
It won't work dynamically now as all shadow receivers would need to recompile their shaders, but I can make it public for construction time.